### PR TITLE
Add pagination; add missing fields and objects [#17]

### PR DESCRIPTION
**Pull request by dnl-blkv**

### CODE REVIEW

**Comment by dnl-blkv** [comment]
@OGKevin when you're done with reviewing, please poke me but don't merge yet! :)

File: BunqSdk/Http/Pagination.cs [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    public class Pagination
+    {
+        /// <summary>
+        /// Error constants.
+        /// </summary>
+        private const string ERROR_NO_PREVIOUS_PAGE =
+            "Could not generate previous page URL params: there is no previous page.";
+
+        /// <summary>
+        /// URL Param constants.
```
Comment (OGKevin) [inline BunqSdk/Http/Pagination.cs]: Ok I triple checked this time haha,  not generated right 😝. Below you have param without capital `P` so I guess this one should also be lower case ?

File: BunqSdk/Json/PaginationConverter.cs [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        private const string FIELD_FUTURE_URL = "future_url";
+
+        /// <summary>
+        /// Indices of param key and value after parsing.
+        /// </summary>
+        private const int INDEX_PARAM_KEY = 0;
+        private const int INDEX_PARAM_VALUE = 1;
+
+        /// <summary>
+        /// Base dummy URL to hack though the incomplete relative URI functionality of dotnetcore.
```
Comment (OGKevin) [inline BunqSdk/Json/PaginationConverter.cs]: Is though a typo ? Shouldn't it be `through`. And this is kinda clever btw 👏

Comment (dnl-blkv) [reply BunqSdk/Json/PaginationConverter.cs]: That's indeed a typo! Thanks :)

File: BunqSdk/Http/Pagination.cs (hunk 1 above)
Comment (dnl-blkv) [reply BunqSdk/Http/Pagination.cs]: Ohh OCD! :D I'll fix this :)

**Comment by OGKevin** [comment]
@dnl-blkv tests passing. please add pagination tests as we discussed and address the changes I proposed and it should be good to merge.

File: BunqSdk/Http/Pagination.cs (hunk 1 above)
Comment (dnl-blkv) [reply BunqSdk/Http/Pagination.cs]: Done

File: BunqSdk/Json/PaginationConverter.cs (hunk 2 above)
Comment (dnl-blkv) [reply BunqSdk/Json/PaginationConverter.cs]: Done

**Comment by dnl-blkv** [comment]
Pagination test to go!

**Comment by OGKevin** [comment]
All tests passing, LGTM. @andrederoos please take a look 👀.

**Pull request merged**